lib/ansible/modules/cloud/docker/docker_node.py

```diff
@@ -0,0 +1,290 @@
+#!/usr/bin/python
+#
+# (c) 2019 Piotr Wojciechowski <piotr@it-playground.pl>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: docker_node
+short_description: Manage Docker Swarm node
+version_added: "2.8"
+description:
+    - Manages the Docker nodes via Swarm Manager
+    - Change node role
+    - Change node availability
+    - Change or remove node labels
+options:
+    state:
+        description:
+            - Set to C(update) to update node labels, availability and role
+        required: true
+        choices:
+          - update
+    name:
+        description:
+            - Swarm name of the node.
+            - Used with I(state=update).
+    labels:
+        description: User-defined key/value metadata.
+    labels_state:
+        description:
+            - Defines the operation on the lables assigned to node.
+            - Set to C(merge) to merge provided labels with already assigned ones. It will update existing keys.
+              The I(labels) must be specified.
+            - Set to C(replace) to replace assigned labels with provided ones. The I(labels) must be specified.
+            - Set to C(remove) to remove all labels assigned to node.
```

Comment:
This is redundant, one can use `replace` with `labels: {}`.

Reply:
Will use `remove` to remove single or multiple labels provided in `labels`

Reply:
Actually I think it may be better to use `remove` for both. If `labels` is not specified then it will remove all lables, otherwise it will remove just labels specified in `labels`

Reply:
I don't think that's a good UI.